src/teleop/gui/src/components/Cameras.vue

```diff
@@ -1,42 +1,36 @@
 <template>
   <div class="wrap">
-    <h3>Cameras</h3>
-    <div class="input">
-      Camera name:
-      <input v-model="cameraName" class="rounded" type="message" /> Camera
-      number:
-      <input
-        v-model="cameraIdx"
-        class="rounded"
-        type="Number"
-        min="0"
-        max="8"
-      />
-      <button class="rounded button" @click="addCameraName()">
-        Change name
-      </button>
+    <div class="block box1">
```

Comment:
These changes are more substantial than the others, did you retest this with the debug_service node